src/main/java/seedu/address/model/food/FoodMethodHandler.java

```diff
@@ -0,0 +1,7 @@
+package seedu.address.model.food;
+
+public class FoodMethodHandler {
+
+    // To add methods to handle total food and return relevant
+
```

Comment:
Maybe you could add a TODO comment on what kinds of methods to add to this class/how it should be used.

e.g. `// TODO: Food FoodMethodHandler.combineFood(Food firstItem, Food secondItem)` or something